home/gettingstarted/0.7/executors.md

```diff
@@ -155,9 +155,40 @@ And the final output:
         (ci = <20000.02 kB, 20000.02 kB>, significance = 1.0E-10)
 
 
-You can take a look at the ScalaMeter API or source code to figure out how different measurers work in more detail.
+### Method Invocation Count
+
+The invocation count measurers are used to count the total number of invocations
+of a specific method, or a group of methods.
+
+There are two implementations of the `InvocationCount` trait and both of them need instrumented classpath to work,
+hence `SeparateJvmsExecutor` must be used as an executor.
+
+`BoxingCount` simply counts (auto)boxing of primitives: `Boolean`, `Byte`, `Char`, `Short`, `Int`, `Long`, `Float` and `Double`.
+You can either specify primitives you would like to count or count all by using `BoxingCount.all()` method.
+
+`MethodInvocationCount` can count invocations of arbitrary methods. Configuration of it is done via specifying `InvocationCountMatcher` constructor argument.
+
+`InvocationCountMatcher` matches class and its methods and selects those that should be instrumented using [ASM](http://asm.ow2.org/). 
+It consists of two parts: `ClassMatcher` and `MethodMatcher`.
 
+#### Class Matcher
+`ClassMatcher` is part of the `InvocationCountMatcher` that selects classes whose methods should be matched using `MethodMatcher`.
 
+- `ClassMatcher.ClassName` matches class by its name
+- `ClassMatcher.Regex` matches class with a regular expression
+- `ClassMatcher.Descendants` matches class over it parents (superclass and mixings). It can either match only direct parents or search through the inheritance tree.
+
+#### Method Matcher
+
+After class selection by the `ClassMatcher` every method is checked against the `MethodMatcher`, which select methods that should be instrumented.
+There are following kinds of the `MethodMatcher`:
+
+- `MethodMatcher.MethodName` that matches method by its name
+- `MethodMatcher.Regex` that matches method with a regular expression 
+- `MethodMatcher.Full` that matches both method name and method descriptor, which is basically the internal format of method arguments and return type.
+
+
+You can take a look at the ScalaMeter API or source code to figure out how different measurers work in more detail.
```

Comment:
Please create a couple of examples in `scalameter-examples` with combinations of these matchers, and link to those examples from here.